reference/dic_tags.rst

```diff
@@ -1360,9 +1360,14 @@ the new loader and tag it with ``twig.loader``:
 
         $container
             ->register('acme.demo_bundle.loader.some_twig_loader', 'Acme\DemoBundle\Loader\SomeTwigLoader')
-            ->addTag('twig.loader')
+            ->addTag('twig.loader', array('priority' => 0))
         ;
 
+.. note::
+
+    The ``priority`` value is optional, and defaults to 0.
```

Comment:
the comma before and is called a Serial comma, which we don't use in the docs. Can you please remove it?

Also, I think it'll be nice if `0` was put in double backticks to make it a literal.